common/sass/styles/overrides/forms.scss

```diff
@@ -33,12 +33,12 @@ input[readonly] {
   margin-top: $space-sm;
   margin-bottom: $space-sm;
   transition: $transition;
+  padding: 0.75rem 16px;
```

Comment:
Make padding(s) a variable to reuse with react-select. Prefer rem over px for better responsive behavior.